### PR TITLE
feat: add get-involved block on home page

### LINKED ISSUE #23: Add a get involved block from figma

### PR DESCRIPTION
**Pull request by divyanshugarg36**
This adds up to the last block of this project solving https://github.com/divyanshugarg36/brl-cad-web/issues/23 issue

### CODE REVIEW

File: src/components/GetInvolved.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <div className="get-involved">
+      <p>
+        <Trans i18nKey="GET_INVOLVED.TITLE">
+          BRL-CAD is <span>backed by open</span> source community
+        </Trans>
+      </p>
+      <Button elementType="a" className="brl-button--small" href={LINKS.IRC_LINK} target="_blank">{t('GET_INVOLVED.BUTTON')}</Button>
```
Comment (theamanjs) [inline src/components/GetInvolved.tsx]: How come this line is so long? Do you have the max-length rule disabled in eslint?

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: src/components/GetInvolved.tsx (hunk 1 above)
Comment (divyanshugarg36) [reply src/components/GetInvolved.tsx]: Fixed

Comment (theamanjs) [reply src/components/GetInvolved.tsx]: Was the rule not added in the eslint file?

Comment (divyanshugarg36) [reply src/components/GetInvolved.tsx]: This rule is usually enabled by default but it has no effect on JSX code, the reason it didn't highlight the error

**Review by theamanjs — APPROVED** [review]

**Pull request merged**